packages/notifications-system/src/emailDriver/index.js

```diff
@@ -0,0 +1,29 @@
+const nodemailer = require("nodemailer");
+
+// async..await is not allowed in global scope, must use a wrapper
+async function main() {
+  // create reusable transporter object using the Gmail SMTP transport
+  const transporter = nodemailer.createTransport({
+    host: "smtp.gmail.com",
+    port: 587,
+    secure: false, // true for 465, false for other ports
+    auth: {
+      user: "testingbookingsystem@gmail.com", // email of the account that will be used to send the emails
+      pass: "BSTestingC92022", // password of the account (could use env variables for better security)
```

Comment:
As you said in the comment, why you are not using env variables for this?

We don't usually upload credentials or sensitive data into the repo.

Reply:
Thanks for the observation! Will make the changes based on your comment, thanks :)